shared/desktop/app/main-window.js

```diff
@@ -62,7 +63,10 @@ export default function() {
 
   const openedAtLogin = app.getLoginItemSettings().wasOpenedAtLogin
   const isRestore = getenv.boolish('KEYBASE_RESTORE_UI', false) || app.getLoginItemSettings().restoreState
-  const hideWindowOnStart = getenv.string('KEYBASE_START_UI', '') === 'hideWindow'
+  // Start not shown by default on Windows
+  const hideWindowOnStart =
```

Comment:
The default is for the environment variable `KEYBASE_START_UI` to be undefined, in which case `hideWindowOnStart` was being set to false before this change